static/src/stylesheets/module/reader-revenue/_epic-reminder.scss

```diff
@@ -0,0 +1,148 @@
+.epic-reminder {
+    ////////////////////////////////////
+    // Prompt to open reminder
+    // Button to close reminder
+    ////////////////////////////////////
+    .epic-reminder__prompt {
+        width: 100%;
+        text-align: right;
+        cursor: pointer;
+    }
+
+    .epic-reminder__prompt-text {
+        @include fs-textSans(5);
+        font-weight: bold;
+        text-decoration: underline;
+        padding-right: 18px;
+    }
+
+    .epic-reminder__close-button {
+        font-size: 30px;
+        cursor: pointer;
+        float: right;
+        margin-top: $gs-baseline * 2;
+
+        svg path {
+            fill: $brightness-7;
+        }
+    }
+
+    ////////////////////////////////////////////////
+    // Logic for opening and closing the reminder
+    ////////////////////////////////////////////////
+    .epic-reminder__reveal-reminder {
+        display: none;
+    }
+
+    .epic-reminder__reveal-reminder:checked ~ .epic-reminder__signup {
+        display: block;
+    }
+
+    .epic-reminder__reveal-reminder:checked ~ .epic-reminder__prompt-label {
+        display: none;
+    }
+
+    .epic-reminder__signup {
+        @include multiline(4, $brightness-86, top);
+        display: none;
+        line-height: initial;
```

Comment:
Not sure if relevant here, but on support-frontend we get messages that using line-height is discouraged.

Reply:
so line-height is set by using the multiline mixin and i am overriding it here with 'initial' (which is essentially saying 'reset to default) for the reason why setting it is discouraged: it limits line height that can be set on children elements.

Reply:
👍